### DISCUSSION TITLE
no unique ID for xiaomi cloud map extractor

### DISCUSSION THREAD

**Discussion by goofy84** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#745] [Q&A]
can anybody tell me how to fix this? With this error message i can not edit the card
![map extractor](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/44529820/6bf27e06-cc6c-4c9d-89dd-7863327f8616)

**Comment by PiotrMachowski** [comment]
This is not this card, this is a [picture entity card](https://www.home-assistant.io/dashboards/picture-entity/). This is just a warning message and it doesn't break anything
